### PR TITLE
5688.dev News Page aded Styles to Top block 

### PR DESCRIPTION
**Pull request by tymoshchuk-o**
https://redmine.devbranch.work/issues/5688

### CODE REVIEW

File: web/themes/custom/green_theme/src/scss/_main-news.scss [hunk 1]
```diff
@@ -0,0 +1,105 @@
+// Styles top news News page
+
```
Comment (Y0r) [inline web/themes/custom/green_theme/src/scss/_main-news.scss:2]: ?

File: web/themes/custom/green_theme/templates/block--block-content--custom-site-logo--full.html.twig [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ *
+ * @see template_preprocess_block()
+ *
+ * @ingroup themeable
+ */
+#}
+
+{%
+ set logoClass=['green-logo', 'green-page-logo']
+ %}
```
Comment (Y0r) [inline web/themes/custom/green_theme/templates/block--block-content--custom-site-logo--full.html.twig:35]: CS

File: web/themes/custom/green_theme/templates/block--block-content--custom-site-logo--full.html.twig [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+ {% block content %}
+   <div{{ content_attributes.addClass(logoClass) }}>
+     <a class="green-logo-link" href="{{ content.field_logo_link[0]['#title'] }}">
+       {{ content.field_site_logo }}
+     </a>
+   </div>
```
Comment (Y0r) [inline web/themes/custom/green_theme/templates/block--block-content--custom-site-logo--full.html.twig:42]: Why?

File: web/themes/custom/green_theme/templates/field/field--node--created--posts.html.twig [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    'field--type-' ~ field_type|clean_class,
+    'field--label-' ~ label_display,
+  ]
+%}
+<span{{ attributes.addClass(classes) }}>
+  {%- for item in items -%}
+    {{ item.contentfield_arrival_date.value | date("F j, o")}}
+
+  {%- endfor -%}
+</span>
```
Comment (Y0r) [inline web/themes/custom/green_theme/templates/field/field--node--created--posts.html.twig:44]: CS

File: web/themes/custom/green_theme/templates/node--posts--top-post.html.twig [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+#}
+{% set classes = [
+  'node',
+  'node--type-' ~ node.bundle|clean_class,
+  node.isPromoted() ? 'node--promoted',
+  node.isSticky() ? 'node--sticky',
+  not node.isPublished() ? 'node--unpublished',
+  view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
+  'clearfix',
+] %}
```
Comment (Y0r) [inline web/themes/custom/green_theme/templates/node--posts--top-post.html.twig:77]: CS

File: web/themes/custom/green_theme/templates/node--posts--top-post.html.twig [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+      <div class="node__meta">
+        <span{{ author_attributes }}>
+          {% trans %}{{ author_name }} - {{ date }}{% endtrans %}
+        </span>
+        {{ metadata }}
+      </div>
+    {% endif %}
+  </header>
+
+</article>
```
Comment (Y0r) [inline web/themes/custom/green_theme/templates/node--posts--top-post.html.twig:105]: CS

File: web/themes/custom/green_theme/templates/paragraph--follow-us-item-block--default.html.twig [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+{% set classes = [
+  'paragraph',
+  'paragraph--type--' ~ paragraph.bundle|clean_class,
+  view_mode ? 'paragraph--view-mode--' ~ view_mode|clean_class,
+  not paragraph.isPublished() ? 'paragraph--unpublished'
+] %}
+
+      <a class="green-social-link" href="{{ content.field_icon_link[0]['#title'] }}">
+        {{ content.field_icon_image }}
+      </a>
```
Comment (Y0r) [inline web/themes/custom/green_theme/templates/paragraph--follow-us-item-block--default.html.twig:50]: CS

**Review by Y0r — CHANGES_REQUESTED** [review]

File: web/themes/custom/green_theme/templates/block--block-content--custom-site-logo--full.html.twig (hunk 3 above)
Comment (tymoshchuk-o) [reply web/themes/custom/green_theme/templates/block--block-content--custom-site-logo--full.html.twig:42]: Different logo from the footer

**Review by Y0r — APPROVED** [review]

**Pull request merged**